lib/compiler/src/beam_trim.erl

```diff
@@ -132,14 +136,14 @@ create_map(Trim, []) ->
        (Any) -> Any
     end;
 create_map(Trim, Moves) ->
-    GbTree0 = [{Src,Dst-Trim} || {move,{y,Src},{y,Dst}} <- Moves],
-    GbTree = gb_trees:from_orddict(sort(GbTree0)),
+    Map0 = [{Src,Dst-Trim} || {move,{y,Src},{y,Dst}} <- Moves],
+    Map = maps:from_list(Map0),
     IllegalTargets = gb_sets:from_list([Dst || {move,_,{y,Dst}} <- Moves]),
```

Comment:
Would it make sense to change this to `cerl_sets` as well?

Reply:
Yes, it does.